lib/toml/string.rb

```diff
@@ -16,3 +16,28 @@ def self.transform_escaped_chars(str)
       .gsub(/\\\\/, '\\')
   end
 end
+
+module TomlLiteralString
+  def value
+    first.value[1...-1]
+  end
+end
+
+module TomlMultilineString
+  def value
+    aux = captures[:text].first.value
+
+    # Remove spaces on multilined Singleline strings
+    aux.gsub!(/\\\r?\n[\n\t\r ]*/, '')
+
+    TomlBasicString.transform_escaped_chars aux
```

Comment:
There's no way to avoid repetition here, right? `value` method has to be declared at `Module#value` level?

I ask because the `TomlMultilineString#value` is just `TomlMultilineLiteral#value` with `TomlBasicString.transform_escaped_chars`.

It would be nice to be able express it that way